src/modules/home/home-utils.js

```diff
@@ -0,0 +1,60 @@
+export const likeClass = 'heart fas fa-heart';
+export const unlikeClass = 'heart far fa-heart';
+export const carRender = (
+  id,
+  imgUrl,
+  make,
+  model,
+  likes = 0,
+) => `<div
+      class="car-card flex flex-col gap-2 border-4 rounded-md border-violet-600" id="${id}">
+      <div class="car-img h-50 object-cover">
+        <img class="w-full"
+          src="${imgUrl}"
+          alt="${make.toUpperCase()} ${model.toUpperCase()}">
+      </div>
+      <div class="car-info flex justify-between p-2 mt-auto">
+        <h2>${make.toUpperCase()} ${model.toUpperCase()}</h2>
+        <div class="flex flex-col items-center text-red-600">
+          <i class="${unlikeClass}"></i>
+          <span class="text-black"><span class="likes">${likes}</span> <span>likes</span></span>
+        </div>
+      </div>
+      <div class="car-buttons flex flex-col gap-3 p-2 ">
+        <button id="model" class="comment-button px-2 py-1 border-2 hover:border-2 hover:shadow-sm hover:text-white hover:bg-sky-500"
+          type="button">Comment</button>
+        <button
+          class="reservation-button px-2 py-1 border-2 hover:border-2 hover:shadow-sm hover:text-white hover:bg-sky-500"
+          type="button">Reservation</button>
+      </div>
+    </div>`;
+
+export function fileExists(imageUrl) {
```

Comment:
- [ ] according to the requirements the Project should use ES6, which means you should avoid usage of the function keyword, please use arrow functions instead.